libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -115,6 +115,25 @@ class AP_AHRS
         return _flags.fly_forward;
     }
 
+    void set_done_takeoff(bool b) {
```

Comment:
Can we change the name of this so is_flying or equivalent. 'Done takeoff' as discussed below is ambiguous. The definition also needs to be documented to make it clear that this is true when the vehicle is flying, not just when it is taking off.